src/content/docs/blog/roadmap-2025.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Roadmap 2025 and Biome 2.0
+summary: "A look at what 2025 will bring for us: Biome 2.0, enterprise support, and our roadmap"
+authors:
+  - arendjr
+  - team
+date: 2025-01-20
+cover:
+  light: "@/assets/blog/roadmap-2024/banner-light.png"
+  dark: "@/assets/blog/roadmap-2024/banner-dark.png"
+  alt: "Biome - Toolchain of the web"
+socialImage: "@/assets/social-logo.png"
+---
+
+Today we're happy to share our plans for Biome 2.0 as well as the rest of our roadmap for 2025. But before we dive into what's coming, let's do a quick recap of the major developments in 2024.
+
+## 🎆 Recap: Biome in 2024
+
+2024 was a great year for Biome. Let's see what happened:
+
+* We released 4 new "minor" Biome versions, from 1.6 through 1.9, with plenty of useful features:
+  * New `biome search` and `biome explain` commands, while the `biome migrate` command was significantly expanded to help users coming from ESLint and Prettier.
+  * Added support for **CSS** and **GraphQL** formatting and linting. 
+  * Partial support for **Astro**, **Svelte** and **Vue** files.
+  * The ability to let configuration files extend from one another, which is especially useful in monorepo and larger organizational setups.
+  * We added countless new lint rules and miscellaneous fixes and improvements.
+ * Our [team of maintainers](https://github.com/biomejs/biome/blob/main/CONTRIBUTING.md#current-members) has grown from 10 members at the start of 2024 to 18 today.
+ * We won the **Productivity Booster** award of the [OS Awards 2024](https://osawards.com/javascript/2024).
+ * We gained several new [sponsors](https://github.com/biomejs/biome#sponsors).
+ * Our VS Code extension has seen an overhaul that's currently in Pre-Release.
```

Comment:
We could mention the Zed extension, too, which was graciously given by https://github.com/luckydye to the Biome org, and they keep maintaining it. It feels like a deserved shout-out!